homework13_4.py

```diff
@@ -0,0 +1,20 @@
+# 4. Написати sql запит, який буде робити те саме, що і в завданні 3, але виводити дані
+# відсортовані по кількості користувачів у спадаючому порядку та по віку у зростаючому
+# порядку. Результатом має бути таблиця:
+    # age  | users
+    # 52    | 2
+    # 120  | 2
+    # 32    | 1
+    # 1142 | 1
+
+import sqlite3
+from pprint import pprint
+
+conn = sqlite3.connect('database.sqlite')
+cursor = conn.cursor()
+
+query = ("SELECT age, COUNT(id) FROM users GROUP BY age ORDER BY COUNT(id), age desc")
```

Comment:
You can use alias here for COUNT(id) and order by the alias: `... COUNT(id) as users ... ORDER BY users.

And pay attention: you should order by users in descending order and by age in ascending